### DISCUSSION TITLE
There seems a bug at Line 398# in dpgen/dpgen/generator/run.py

### DISCUSSION THREAD

**Discussion by ZhouXY-PKU** [deepmodeling/dpgen#504] [General]
Hello,
I accidentally found that Line **397#** & **398#** of the **master branch of dpgen/dpgen/generator/run.py** is all the same, and I guess that '**descriptor**' at Line **398#** should be change to '**fitting_net**'.
Is that true? Thank you very much!

**Comment by njzjz** [comment]
Already fixed by https://github.com/deepmodeling/dpgen/pull/484.

**Reply by ZhouXY-PKU** [reply]
OK, thank you very much!
